cloud-stream-sink-null-payload-converter/src/main/resources/application.yml

```diff
@@ -0,0 +1,38 @@
+spring:
+  cloud:
+    function:
+      definition: sink
+    stream:
+      bindings:
+        sink-in-0:
+          destination: TEMPS.Q
+          #The presence of "group" tells the binder to follow the "consumer group" pattern
+          group: SINK
+          consumer:
+            #Concurrency can be used with a consumer group (non-exclusive queue) to process events in multiple threads
+            concurrency: 5
+      binders:
+        local-solace:
+          type: solace
+          environment:
+            solace:
+              java:
+                host: tcp://localhost:55555
+                msgVpn: default
+                clientUsername: default
+                clientPassword: default
+      solace:
+        bindings:
+          sink-in-0:
+            consumer:
+              #This adds a topic subscription w/ wildcards to the queue created with a name of TEMPS.Q.SINK above 
+              queueAdditionalSubscriptions: sensor/temperature/>
+                
+logging:
+  level:
+    root: INFO
+    com.solacesystems.jcsmp: INFO
+    org.springframework: INFO
+    
+server:
```

Comment:
remove if not needed

Reply:
Sure, removed. In fact, removed from other application.yaml files as well.